client/src/Components/JournalPrompt/JournalPrompt.js

```diff
@@ -1,17 +1,30 @@
 import React, {useState} from "react";
-import { Link } from "react-router-dom";
+import { useHistory } from "react-router-dom";
 import PropTypes from 'prop-types';
 import './JournalPrompt.css';
 
 const JournalPrompt = ( {updateJournal, feeling} ) => {
   const [newEntry, setNewEntry] = useState('');
+  const [error, setError] = useState(false);
+  const history = useHistory();
+
+  const handleClick = () => {
+    if (newEntry) {
+      updateJournal(newEntry);
+      history.push(`/what-should-you-do/${feeling}`)
+    } else {
+      setError(true);
+      setTimeout(() => {setError(false)}, 2000);
+    }
+  }
 
   return (
     <div className="prompt-container">
       <form>
         <h2 className="prompt">Why are you feeling this way?</h2>
-        <textarea id="journal-entry" onChange={(event) => setNewEntry(event.target.value)}></textarea>
-        <Link to={`/what-should-you-do/${feeling}`} className="uni-btn" onClick={() => updateJournal(newEntry)}>→</Link>
+        <textarea required id="journal-entry" onChange={(event) => setNewEntry(event.target.value)}></textarea>
+        <p className="uni-btn" onClick={handleClick}>→</p>
+        {error ? <p style={{textAlign: 'center'}}> Please tell us a bit more about how you're feeling today - you can always edit it later. 🤍 </p> : null}
```

Comment:
i ❤️ this a lot

Reply:
Huge yes to letting the user know they can edit later. I think thats super helpful for UX